app/models/agents/java_script_agent.rb

```diff
@@ -102,6 +102,7 @@ def execute_js(js_function, incoming_events = [])
           memory.to_json
         end
       end
+      context["unescapeHTML"] = lambda { |a, x| CGI.unescapeHTML(x) }
```

Comment:
According to MDN, unescape is deprecated? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape

I'm not entirely familiar with javascript - I wasn't even aware unescape() was a thing when I was writing this. Every solution I found required either a nodejs library, or the DOM to be available, which presumably isn't a thing for either.

After testing this on a local installation (chrome first, then huginn), unescape doesn't unescape html characters at all.